_posts/2021-01-25-roadmap.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: Search.gov Roadmap
+crumbname: Roadmap
+category: about
+tags: roadmap
+breadcrumbs:
+  - label: Home
+    url: /
+  - label: About
+---
+
+## Overview
+
+At Search.gov we aim to provide an excellent public search experience, a self-service search administration experience, and a rock-solid secure and compliant infrastructure to support it all, while keeping costs down across the government.
+
+We know it’s helpful for our partners at federal agencies to know what’s coming up next, to support planning for your own web platforms. 
+
+This roadmap shows what we are working on now, and what we plan to be working after that. We’ll update this roadmap quarterly.
+
+
+## FY21, Q2: January - March 2021
+
+### Now
+
+* **Version Upgrades**
+  * We will be upgrading our Elasticsearch and Ruby components to keep them up-to-date and compliant. 
+* **Search.gov website redesign**
+  * We will be launching a new design that aligns with the [US Web Design System](https://designsystem.digital.gov/), in accordance with the [21st Century IDEA](https://digital.gov/resources/21st-century-integrated-digital-experience-act/). We are excited for your feedback!
+* **Search.gov Search Results Page (SERP) Redesign**
+  * We will begin the early stages of scoping out the work required to upgrade the SERP and decommission out-of-date variants of the out-of-the-box experience.
```

Comment:
@amanifarooque can we lighten this up? Maybe just use the language from the last roadmap?

Reply:
Copied the text for the website redesign, and slimmed down the SERP redesign bullet!